packages/forms/src/UIForm/Widget/Widget.component.js

```diff
@@ -7,15 +7,31 @@ import { getError } from '../utils/errors';
 import { getValue } from '../utils/properties';
 import shouldRender from './condition';
 
+function getWidget(displayMode, widgetId, customWidgets) {
+	// resolve the widget id depending on the display mode
+	let id = widgetId;
```

Comment:
You can use a ternary here to avoid using `let`
`const id = displayMode ? \`${widgetId}_${displayMode}\` : widgetId;`